message_ix_community_meeting.rst

```diff
@@ -0,0 +1,82 @@
+.. _messageix workshop:
+
+MESSAGEix Community Meeting
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Community Meeting aims to bring together researchers working on integrated assessment and/or energy system modeling using open source software. 
+
+It is a platform for showcasing new developments and features of the MESSAGEix modeling framework, sharing knowledge on various modeling approaches and applications of the framework, and discussing best practices in model-based policy analysis. The meeting is a place for facilitating collaboration between MESSAGEix users from around the world, by creating a forum where users can share their latest research outcomes and modeling experiences.
+
+The Community Meeting also aims to inspire new users of the MESSAGEix framework on the range of available possibilities and link them with other users in order to help them advance their modeling work.  
+For information about the model, its structure, mathematical
+formulation and much more, please see the `documentation <https://docs.messageix.org>`_.
+
+.. contents:: Table of Content
+   :local:
+
+Upcoming events
+""""""""""""""""
+
+* 24-25 May 2023 - MESSAGEix Community Meeting (`more info and the agenda <https://iiasa.ac.at/events/may-2023/messageix-community-meeting-2023>`_)
+
+Past workshops and seminars
+""""""""""""""""
+
+* 7-8 May 2022 - 1st Annual MESSAGEix Community Meeting (`more info and the agenda <https://iiasa.ac.at/events/may-2022/messageix-community-meeting>`_)
+
+Learning objectives
+"""""""""""""""""""
+
+The learning objectives of the workshops are the following:
+
+* Connecting with other community members.
+* Exploring/Showcasing new applications of MESSAGEix.
+* Getting to know recent advances within MESSAGEix.
+* Understanding more complex aspects of MESSAGEix.
+* Gaining insights on interplay with related tools.
+
+Needed requirements
+"""""""""""""""""""
+
+Each workshop is designed to be accessible for users with different
+backgrounds and levels of experience with the modelling. However, there
+are some pre-requisite knowledge and skills which you should
+go through before getting started, including:
+
+* Elementary computer programming (preferably in the Python or R language);
+* Fundamental concepts of mathematical modelling, optimization,
+  and data analysis; and
+* Energy systems (e.g., energy supply, energy conversion
+  technologies, and demand sectors and their linkages).
+
+For a complete list, plus links to learning resources, see
+`“Pre-requisite knowledge & skills" repo <https://docs.messageix.org/en/stable/prereqs.html>`_
+in the documentation.
+
+Installation of MESSAGEix
```

Comment:
As mentioned my fist comment above, we can remove this Section and refer to the installation pages in the first section.